packages/bits/demo/src/components/demo/dialog/dialog-docs/dialog-docs.example.component.html

```diff
@@ -131,6 +131,10 @@ <h2>Dialog Inside Overlay</h2>
     <nui-dialog-inside-overlay-example></nui-dialog-inside-overlay-example>
 </nui-example-wrapper>
 
+<nui-example-wrapper filenamePrefix="dialog-inside-overlaywith-date-time-picker" exampleTitle="Dialog Inside Overlay With DateTimePicker Component">
```

Comment:
I don't think we need an example in the docs for this since it's only a bugfix. We should make sure anything that goes in the docs is added there for the purpose of increasing the understanding of our consumers.

For the e2e tests, the best thing to do in this case will be to create a separate test component that instantiates all the same components that are instantiated in this existing DialogDocsComponent. It can be called DialogTestComponent. The "dialog-test" route found [here](https://github.com/solarwinds/nova/blob/f1291391005ea3e52a332abd99ce70729af1ef36/packages/bits/demo/src/components/demo/dialog/dialog.module.ts#L57) can point to this new DialogTestComponent instead of the docs one.

Reply:
Created a separate test component, thanks